homeassistant/helpers/template.py

```diff
@@ -325,14 +330,15 @@ def render(
 
         return run_callback_threadsafe(
             self.hass.loop,
-            partial(self.async_render, variables, parse_result, **kwargs),
+            partial(self.async_render, variables, parse_result, limited, **kwargs),
         ).result()
 
     @callback
     def async_render(
         self,
         variables: TemplateVarsType = None,
         parse_result: bool = True,
+        limited: bool = False,
```

Comment:
Can you document what `limited` does in the doc string.

Reply:
Done!